scripts/amount-modifier.js

```diff
@@ -1,19 +1,29 @@
 var currencies = require('../data/currencies.js');
 
 var amountModifier = {
+  convertToCents: function(amount) {
```

Comment:
I think this should be an internal API only, not something outside users can access